addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -631,6 +632,7 @@ def test_message_process_crash_no_data(self):
     def test_message_process_crash_alien_domain_same_alias(self):
         """Incoming email to the same address in an alien domain must raise."""
         self.env['ir.config_parameter'].set_param('mail.catchall.domain', 'example.com')
+        self.env['ir.config_parameter'].set_param('mail.catchall.domain.strict', True)
```

Comment:
@MiquelRForgeFlow As you can see, the problem in tests was that they were testing a behavior that actually was disabled. The fact they were failing was actually good. I just enabled it and now it works fine.

Reply:
ah, good